### PR TITLE
[vim/de] attempt to fix broken build for vim de

### PR DESCRIPTION
**Pull request by mxschumacher**
- [X ] I solemnly swear that this is all original content of which I am the original author
- [ X] Pull request title is prepended with `[language/lang-code]`
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by mxschumacher** [comment]
I've had a stale fork (500+ commits behind master) when I made this change. I merged all the new changes from the origin repo into my fork, but now there are two commits (the merge and the change I made). Is there a cleaner way to do this? I could have deleted my fork and create a new one, but that seemed to drastic

**Comment by imba-tjd** [comment]
```bash
git reset --hard HEAD~2
git pull upstream
git cherry-pick 0efd9d8f13f261846b27f0df0fd1f482e440ba4e
```

**Comment by kevinnls** [comment]
ah this PR got hidden by some filtering my PR #4001 is a duplicate
alsoo on a remotely related but rather unrelated not check out my [gist](https://gist.github.com/kevinnls/c4050c208723427c45ef3924d6d3bc5a) for a trick i found for keeping forks synced

File: de-de/vim-de.html.markdown [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 category: tool
 tool: vim
-lang = de-de
+lang : de-de
```
Comment (kevinnls) [inline de-de/vim-de.html.markdown:4]:
```suggestion
lang: de-de
```
just the minor formatting

**Comment by mxschumacher** [comment]
Thanks for the help with git @imba-tjd  . I'll close this PR and accept yours, @kevinnls

**Pull request closed without merging**